usefulUtilities/scaleAvatar/scaleAvatar.js

```diff
@@ -0,0 +1,135 @@
+//
+//  Created by Robin Wilson on 3/11/2019
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+(function () {
+    var DEBUG = false;
+
+    // Returns boolean that all buttons are pressed
+    var isPressed = {
+        rightGrip: false,
+        leftGrip: false,
+        rightTrigger: false,
+        leftTrigger: false
+    };
+    function areAllPressed() {
+        return isPressed.rightGrip && isPressed.leftGrip && isPressed.rightTrigger && isPressed.leftTrigger;
+    }
+
+
+    // Sets up the scaling functionality if all buttons are pressed
+    var SCALE_BUFFER = 1, // increase/decrease the sensitivity of scaling
+        PRESSED_INTERVAL_MS = 25, 
+        isScaling = false, // the avatar is scaling after triggers and grips are pressed
+        initialDistance = null,
+        initialScale = null,
+        scalingInterval = null;
+    function handleAllPressed() {
+        if (areAllPressed() && isScaling) {
+            // setup was completed already, do not need to continue
+            return;
+        } else if (areAllPressed() && !isScaling) {
+            // Initial time isScaling called
+            initialDistance = distanceBetweenHands();
+            isScaling = true;
+            initialScale = MyAvatar.scale;
+
+            // Setup pressed interval
+            scalingInterval = Script.setInterval(function() {
+                if (isScaling) {
+                    if (DEBUG) {
+                        print("init" + JSON.stringify(initialDistance));
+                        print("distance" + JSON.stringify(distanceBetweenHands()));
+                    }
+                    var currentDistance = distanceBetweenHands();
+                    var deltaDistance = currentDistance - initialDistance;
+                    MyAvatar.scale = initialScale + deltaDistance * SCALE_BUFFER;
+                }
+            }, PRESSED_INTERVAL_MS);
+        } else {
+            // areAllPressed is false 
+            // we need to stop scaling and remove the scalingInterval
+            if (scalingInterval) {
+                if (DEBUG) {
+                    print("CLEAR INTERVAL");
+                }
+                Script.clearInterval(scalingInterval);
+                scalingInterval = null;
+                isScaling = false;
+                initialDistance = null;
+            }
+        }
+    }
+
+
+    // Get the distance between the avatar's hands
+    function distanceBetweenHands() {
+        var rightHand = Controller.getPoseValue(Controller.Standard.RightHand).translation;
+        var leftHand = Controller.getPoseValue(Controller.Standard.LeftHand).translation;
+        return Vec3.distance(rightHand, leftHand) / MyAvatar.sensorToWorldScale;
+    }
+
+
+    // With specific controller button, sets up controller pressed callback
+    var PRESSED_MIN_VALUE = 0.8;
```

Comment:
this seems a little low - is there a reason you didn't set it to something very high like `0.95` ?